pkg/skopeo/client.go

```diff
@@ -89,7 +89,12 @@ func (c *client) Inspect(image string) (*DockerImage, error) {
 		args = append(args, "--creds", c.dockerAuthConfigs[i].Username+":"+c.dockerAuthConfigs[i].Password)
 	}
 
-	args = append(args, "--config", "docker://"+image)
+	config := "docker://" + image
+	if registry != "" {
+		config = registry + "/" + image
```

Comment:
It may happen that `image` has already a registry, like `ghcr.io/some/image` - probably it should ignore `registry` variable then?

Reply:
i guess, they should be stored in private registry as `ghcr.io/hlesey/busybox` in this case, right @ypoplavs ?

Reply:
The path for private registry is: `LOCATION-docker.pkg.dev/PROJECT-ID/REPOSITORY/IMAGE`

Reply:
Test by @ypoplavs 
```yes, I pushed a test image to the registry in testkube project, the path is:
us-east1-docker.pkg.dev/testkube-328312/test/quickstart-image
```